gotools_settings.py

```diff
@@ -77,8 +77,13 @@ def load_goenv():
 
   # Find the go binary on PATH, and abort initialization if it can't be found.
   gobinary = None
-  for segment in ospath.split(':'):
```

Comment:
I should have used https://docs.python.org/3.0/library/os.html#os.pathsep, which would avoid the platform specific code you introduced for this.